src/main/java/com/wavefront/service/FirehoseToWavefrontProxyConnector.java

```diff
@@ -43,10 +43,10 @@ public void connect() {
             firehoseProperties.getParallelism()));
 
     this.dopplerClient.firehose(
-            FirehoseRequest.builder().subscriptionId(firehoseProperties.getSubscriptionId()).build())
-            .subscribeOn(Schedulers.newParallel(WAVEFRONT_FIREHOSE_NOZZLE, firehoseProperties.getParallelism()))
-            .filter(envelope -> filterEventType(envelope.getEventType()))
-            .subscribe(envelope -> proxyForwarder.forward(envelope));
+            FirehoseRequest.builder().subscriptionId(firehoseProperties.getSubscriptionId()).build()).
+            subscribeOn(Schedulers.newParallel(WAVEFRONT_FIREHOSE_NOZZLE, firehoseProperties.getParallelism())).
```

Comment:
indeting is odd, 47-49 should be one level in (check your ide formatting settings)